lively.ide/components/helpers.js

```diff
@@ -34,7 +34,11 @@ export function getEligibleSourceEditorsFor (modId, modSource) {
     }
     return true;
   });
-  return qualifiedBrowsers.map(browser => browser.get('source editor'));
+  // also consider plain text morphs that have a eval environment that matches
+  const qualifiedEditors = $world.withAllSubmorphsSelect(m => {
+    return m.isText && m.editorPlugin?.evalEnvironment?.targetModule === modId;
+  });
+  return arr.uniq([...qualifiedEditors, ...qualifiedBrowsers.map(browser => browser.get('source editor'))]);
```

Comment:
Why do we merge those? Shouldn't everything in the second array always also be part of `qualifiedEditors`?